app/views/events/spread.html.erb

```diff
@@ -82,28 +82,25 @@
       </tr>
       </thead>
 
-      <tbody>
-      <% @presenter.effort_times_rows.each do |row| %>
-        <tr id="<%= "effort_#{row.id}" %>">
-          <td><%= "#{row.display_overall_rank}/#{row.display_gender_rank}" %></td>
-          <td><%= row.bib_number %></td>
-          <td class="text-nowrap"><strong><%= link_to row.full_name, effort_path(row.effort) %></strong></td>
-          <td class="text-nowrap"><%= row.bio_historic %></td>
-          <td class="text-nowrap"><%= row.flexible_geolocation %></td>
-          <td class="text-nowrap"><%= row.effort_status %></td>
-          <% spread_relevant_elements(row.time_clusters).each do |cluster| %>
-            <td class="text-nowrap text-center">
-              <%= time_cluster_display_data(cluster, @presenter.display_style) %>
-            </td>
+      <% @presenter.grouped_effort_times_rows.each do |status_text, rows| %>
+        <tbody>
+          <% if rows.any? %>
+            <tr class="status"> <th colspan="999" scope="rowgroup"> <%= status_text %> </th> </tr>
```

Comment:
I recently learned that you can do `colspan="100%"`. Not sure if that's a huge improvement, but thought it's an interesting way to do it.

Reply:
Looks like this is the same as colspan="100". I did change it, and 100 _should_ be fine (though could imagine 'checkpoints' via GPS tracker every mile for a > 100+ mile race?). But that's an extreme edge case and it'll still span almost all the columns.